src/Firebase.Auth/project.json

```diff
@@ -0,0 +1,11 @@
+﻿{
+  "supports": {},
+  "dependencies": {
+    "Microsoft.NETCore.Portable.Compatibility": "1.0.1",
+    "NETStandard.Library": "1.6.0",
```

Comment:
The target framework is 1.1. The standard library is for compatibility purposes between the old runtime and the 1.6